src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java

```diff
@@ -66,28 +66,27 @@ private static void setupDrivers(Application application) {
 
         DrawPanelController drawerController = DrawerFeature.getDrawerController();
 
+        // Driver for Line Simulator
+        Job2dDriver driver = new LineDriverAdapter(drawerController, LineFactory.getBasicLine(), "basic");
+        DriverFeature.addDriver("Line Simulator", driver);
+        DriverFeature.getDriverManager().setCurrentDriver(driver);
+
         // TransformationDriver for Line Simulator
-        TransformingDriver commonTransformedDriver = new TransformingDriver(new LineDriverAdapter(drawerController, LineFactory.getBasicLine(), "basic"));
-        commonTransformedDriver.addModifier(new ScalingModifier(1.0, 1.0));
-        commonTransformedDriver.addModifier(new RotationModifier(0));
-        commonTransformedDriver.addModifier(new XAxisMovingModifier(0));
-        commonTransformedDriver.addModifier(new YAxisMovingModifier(0));
-        //commonTransformedDriver.addModifier(new VerticalFlipModifier());
-        //commonTransformedDriver.addModifier(new HorizontalFlipModifier());
-
-        DriverFeature.addDriver("Line Simulator", commonTransformedDriver);
+        TransformingDriver commonTransformedDriver = new TransformingDriver(driver);
+        commonTransformedDriver.addModifier(new ScalingModifier(0.25, 0.25));
+        DriverFeature.addDriver("Line Simulator  - transformed", commonTransformedDriver);
         DriverFeature.getDriverManager().setCurrentDriver(commonTransformedDriver);
```

Comment:
do not setCurrentDriver here. Only one can be set at the same time. The default choice is in line 72.